contracts/commands/BaseBalancerFlashLoanCommand.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+/// BaseAAveFlashLoanCommand.sol
+
+// Copyright (C) 2023 Oazo Apps Limited
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.0;
+
+import { ICommand } from "../interfaces/ICommand.sol";
+
+import { ILendingPool } from "../interfaces/AAVE/ILendingPool.sol";
+
+import { IERC20 } from "@openzeppelin/contracts/interfaces/IERC20.sol";
+import { IServiceRegistry } from "../interfaces/IServiceRegistry.sol";
+import { IFlashLoanRecipient } from "../interfaces/Balancer/IFlashLoanRecipient.sol";
+import { ReentrancyGuard } from "@openzeppelin/contracts/security/ReentrancyGuard.sol";
+
+abstract contract BaseBalancerFlashLoanCommand is ICommand, IFlashLoanRecipient, ReentrancyGuard {
+    IServiceRegistry public immutable serviceRegistry;
+    ILendingPool public immutable lendingPool;
+
+    address public trustedCaller;
```

Comment:
This is never written to in this contracts. There is just one `require` (which is commented out). If the require is removed, please remove this one. If the require stays and this value needs to be filled in, I'd suggest adding an internal function `_setTrustedCaller` that can be used in subsequent contracts